app/src/main/java/co/tinode/tindroid/UiUtils.java

```diff
@@ -19,6 +19,7 @@
 import android.graphics.drawable.AnimationDrawable;
 import android.graphics.drawable.Drawable;
 import android.graphics.drawable.LayerDrawable;
+import android.media.ExifInterface;
```

Comment:
The same question: why are you replacing androidx.exifinterface.media.ExifInterface with android.media.ExifInterface?